policy/modules/system/systemd.te

```diff
@@ -964,6 +964,7 @@ optional_policy(`
         dbus_system_bus_client(systemd_timedated_t)
         dbus_connect_system_bus(systemd_timedated_t)
         dbus_read_pid_sock_files(systemd_timedated_t)
+        dbus_watch_pid_dirs(systemd_timedated_t)
```

Comment:
The AVC and the content is for systemd-timedated; the commit message specifies systemd-timesyncd though.